### ISSUE TITLE
"Don't Save" & "Cancel" text is not visible once they get keyboard focus in Unsaved Changes window in both modes of High Contrast.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#385]
### Steps to Reproduce

1. Change the High contrast standards before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Type something in code editor cell in workbooks window. Now press "ALT+F4". It invokes "Unsaved Changes" window.
4. Navigate to "Don't Save" and "Cancel" button in both modes of high contrast.

### Expected Behavior
"Don't Save" and "Cancel" button text should be visible.
### Actual Behavior
"Don't Save" & "Cancel" text is not visible once they get keyboard focus in Unsaved Changes window in both modes of High Contrast.
### User Impact
User cannot see the text.
### Screenshots
![22 hcb](https://user-images.githubusercontent.com/22956130/40763936-7416baa2-64c4-11e8-866f-31c30430a992.png)
![22 hcw](https://user-images.githubusercontent.com/22956130/40763937-7443d848-64c4-11e8-9eb5-be37341473fe.png)
![22 normal mode](https://user-images.githubusercontent.com/22956130/40763938-746de642-64c4-11e8-8ea9-3df2ac7224e4.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
